opencti-platform/opencti-graphql/src/modules/notification/notification-resolver.ts

```diff
@@ -32,6 +33,11 @@ const notificationResolvers: Resolvers = {
   },
   Trigger: {
     triggers: (trigger, _, context) => triggersGet(context, context.user, trigger.trigger_ids),
+    currentUserAccessRight: (trigger, _, context) => getUserAccessRight(context.user, trigger),
+  },
+  TriggerFilter: {
+    user_ids: 'authorized_members.id',
```

Comment:
Why we don't have directly authorized_members.id in filter key ?

Reply:
because `authorized_members` is not functionally comprehensible